ensure-is-media.js

```diff
@@ -10,6 +10,7 @@ let makeUseIsMedia = (
 // https://github.com/viewstools/morph/blob/master/ensure-is-media.js
 
 // based on https://github.com/streamich/use-media
+import '@react-native-extra/match-media'
```

Comment:
what about adding this to a target specific file like we do in `ensure-flow.js`?

Reply:
Sure, will fix it. I just thought it may work fine for all targets with that polifills :)

Reply:
Yeah, should be alright but this way we're adding one dependency less.
Another alternative instead of duplicating the code could be to add the content for the dependency depending on the target in this function. What do you think? Btw, iirc, `state` has `as`

Reply:
That's right, looks like the state has the `as` property so will make use of it there. Thanks for suggestion